models/intel/smartlab-object-detection-0001/README.md

```diff
@@ -0,0 +1,46 @@
+# smartlab-object-detection-0001
+
+## Use Case and High-Level Description
+
+This is a smartlab object detector that is based on YoloX for 416x416 resolution.
+
+## Example
+
+![](./assets/frame0001_top1.jpg)
+
+## Specification
+
+Accuracy metrics obtained on Smartlab validation dataset with yolox adapter for converted model.
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| [COCO mAP (0.5:0.05:0.95)]      | 37.12%                                    |
+| GFlops                          | 1.077                                     |
+| MParams                         | 0.8908                                    |
+| Source framework                | PyTorch\*                                 |
+
+## Inputs
+
+Image, name: `images`, shape: `1, 1, 416, 416` in the format `B, C, H, W`, where:
+
+- `B` - batch size
+- `C` - number of channels
+- `H` - image height
+- `W` - image width
+
+Expected color order is `BGR`.
+
+## Outputs
+
+The net outputs blob with shape: `1, 1, 200, 7` in the format `1, 1, N, 7`, where `N` is the number of detected
```

Comment:
it is not yolox format of output and does not have any relation to your model output
If you do not know, how to describe model output format, please use this template 
https://github.com/openvinotoolkit/open_model_zoo/tree/master/models/public/yolox-tiny#converted-model-1

Reply:
thanks.

fixed the description